src/app/component/Image/Image.style.scss

```diff
@@ -40,6 +40,10 @@
         mask: radial-gradient(ellipse, black 30%, transparent 61%, transparent 100%), radial-gradient(circle, black 24%, transparent 60%, transparent 100%);
     }
 
+    &_isLoaded {
+        background: transparent;
```

Comment:
Not sure this is the best way. Can you please check, wether the homepage slider and cms blocks work and do not blink? There should be smooth transition from grey to image, without any blinks. Call me when you will be checking.